py3status/modules/xrandr_toggle.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""
+Toggle monitor output from the status bar using xrandr.
+
+Configuration parameters:
+    cache_timeout: how often we refresh usage in seconds (default: 10s)
+    distext: custom text or symbol for disconnected status
```

Comment:
I am not saying there shouldn't be a 79 characters limit for other languages, just really annoyed with that limit for languages forcing you to indent at every declaration or loop such as Python. It makes it really easy to pass that number of characters if you have explicit variable and function names.

For example with a 4 space characters indentation, just declaring a class, making a function and having a try catch with a loop or condition and you are already left with 63 characters per line. If on top of that you have a couple explicit names you reach quickly that limit.

In other languages where indentation is not forced on you it's much easier to deal with.
This is of course my personal opinion about Python, flake8 and the 79 characters limitation.